src/formatDate.js

```diff
@@ -8,7 +8,26 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
```

Comment:
Using `Array.prototype.pop` modifies the original `fromFormat` array, which could lead to unexpected behavior if the array is used elsewhere after calling this function. Consider using a different approach that doesn't mutate the input array.